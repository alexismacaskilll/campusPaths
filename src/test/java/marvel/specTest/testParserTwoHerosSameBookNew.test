# Test the parser on a file that contains two heros in the same book

# Load a graph from tsv file that contains two heros in the same book
LoadGraph twoHerosSameBook twoHerosSameBookNew.tsv

# Print the heros in the graph
ListNodes twoHerosSameBook

# Print the outgoing edges from hero 1
ListChildren twoHerosSameBook hero_1
# Print the outgoing edges from hero 2
ListChildren twoHerosSameBook hero_2