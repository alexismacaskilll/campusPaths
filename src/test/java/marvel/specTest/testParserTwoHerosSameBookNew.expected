# Test the parser on a file that contains two heros in the same book

# Load a graph from tsv file that contains two heros in the same book
loaded graph twoHerosSameBook

# Print the heros in the graph
twoHerosSameBook contains: hero 1 hero 2

# Print the outgoing edges from hero 1
the children of hero 1 in twoHerosSameBook are: hero 2(book 1)
# Print the outgoing edges from hero 2
the children of hero 2 in twoHerosSameBook are: hero 1(book 1)